frontend/src/actions/api/embedding.ts

```diff
@@ -0,0 +1,31 @@
+import axios from "axios";
+
+const BASE_URL = "http://127.0.0.1:8000/api";
+
+export const getEmbeddingPoints = async (): Promise<
+  Array<[number, number, number]>
+> => {
+  try {
+    const response = await axios.get(`${BASE_URL}/embeddings`);
+    return response?.data?.data || [];
+  } catch (error) {
+    return [];
```

Comment:
Let's display an error to the user. A simple approach is create a global error state variable and when set it displays a popup of some sort to report the error.

Reply:
Here you can just throw the error: `throw new Error(error)`